mini-e2e.groovy

```diff
@@ -1,8 +1,7 @@
 def cico_retries = 16
 def cico_retry_interval = 60
-// temporary git repository for testing purpose
-def ci_git_repo = 'https://github.com/nixpanic/ceph-csi'
-def ci_git_branch = 'mini-e2e'
+def ci_git_repo = 'https://github.com/ceph-csi/ceph-csi'
```

Comment:
it should be `https://github.com/ceph/ceph-csi` if yes can we reuse  the git_repo?

Reply:
yes, of course!

Reply:
In case the script is used for a different repo, using the same variable may make it fail.